### DISCUSSION TITLE
Automatisches Löschen der Timesheets

### DISCUSSION THREAD

**Discussion by ghost** [kimai/kimai#2909] [General]
Hey Kevin, da wir für unsere Firma Kimai nutzen wollen, bräuchten wir eine Funktion, womit die Timesheets automatisch nach einer gewissen Zeit gelöscht werden.. Bisher habe ich so eine Funktion nicht gefunden und in der Datenbank rum zu pfuschen wäre auch nicht so optimal.. Gibt es da vielleicht etwas, das ich übersehen habe? Oder könnte so eine Funktion in absehbarer Zeit kommen?

Mit freundlichen Grüßen,

leodre

**Comment by kevinpapst** [comment]
Das kann man über die API automatisieren. Zuerst eine Anfrage um alle Zeiten zu bekommen, die älter sind als X Tage und im Anschluß diese Zeiten dann einfach löschen.

Wenn Ihr das über Kimai selber machen wollt, könnt Ihr mich gerne beauftragen um ein Plugin dafür zu bauen.

**Reply by ghost** [reply]
Okay, danke für die Antwort. Ich würde es dann mal über die API probieren, aber mal so aus Neugier, wie viel würde uns das denn ungefähr kosten, wenn du dafür ein Plugin bauen würdest?

**Reply by kevinpapst** [reply]
Bitte dazu eine Email schreiben, wenn echtes Interesse besteht 😃
